docs/configuration.md

```diff
@@ -792,14 +790,16 @@ EDNS Client Subnet (ECS) configuration parameters:
 
 ## Special Use Domain Names
 
-SUDN (Special Use Domain Names) are always enabled as they are required by various RFCs.  
+SUDN (Special Use Domain Names) are always enabled by default as they are required by various RFCs.  
 Some RFCs have optional recommendations, which are configurable as described below.
+However, you can completely deactivate the blocking of SUDN by setting enable to false
```

Comment:
There should be a warning included that this should only be done if your upstream DNS server is local since this shouldldn't be disabled for remote upstreams.

Reply:
Add a warning as desired